src/Effects.js

```diff
@@ -581,6 +581,7 @@ module.exports = class Effects {
       },
       update: function () {},
       draw: function ({bpm}) {
+        bpm = bpm || 120;
```

Comment:
Maybe worth a comment explaining why this needs a fallback?

Reply:
Added!